alcs-api/src/comment/comment.controller.spec.ts

```diff
@@ -78,7 +71,7 @@ describe('CommentController', () => {
     const comments = await controller.get('file-number', request);
 
     expect(comments.length).toEqual(1);
-    expect(comments[0].author).toEqual(user.name);
+    expect(comments[0].author).toStrictEqual(comment.author.name);
```

Comment:
Why do you use strictEqual? Is it just habit?

Reply:
I noticed that the regular equal could complain even if expected and actual results are the same. So strict equal just a way to compare the context of objects and leave aside references etc.

Reply:
According to documentation strict equal does the same check as isEqual but is stricter in 3 cases.
https://jestjs.io/docs/expect#tostrictequalvalue
Are you confusing toEqual and toBe?